test/unit/coupling/paralleltopology/XYZTopologyTest.cpp

```diff
@@ -17,8 +18,75 @@ class XYZTopologyTest : public CppUnit::TestFixture
   void tearDown() {
   }
 
+   void testGetProcessCoordinates3D() {
+    unsigned int nx = 2;
+    unsigned int ny = 3;
+    unsigned int nz = 5;
+    tarch::la::Vector<3, unsigned int> numberOfProcesses{nx, ny, nz};
+    unsigned int topologyOffset = 10;
+    coupling::paralleltopology::XYZTopology<3> topology{numberOfProcesses, topologyOffset};
+    for (unsigned int rank = topologyOffset; rank < topologyOffset + nx*ny*nz; ++rank) {
+      tarch::la::Vector<3, unsigned int> processCoordinates = topology.getProcessCoordinates(rank);
+      unsigned int z = (rank-topologyOffset)/(nx*ny);
```

Comment:
This is correct, but the comment in [ZYXTopology.h](https://github.com/HSU-HPC/MaMiCo/pull/35/files#diff-e0252921926fb82d01c894932d53950a1db66466b5a8138eb99893edaf8c3ebc) line 23 is wrong, so now there is a mismatch between what the XYZTopology unit test is doing and the documentation / the description about what XYZTopology is supposed to do.

I think @jafariv had confused the two files and put the comment for  XYZTopology in the ZYXTopology.cpp file and vice versa. @LouieVoit can you please double-check and correct the comments (in XYZTopology.cpp and ZYXTopology.cpp, lines ca 21 - 26 in both files) before merging this PR? Otherwise the new unit tests are very confusing.

Reply:
Done